processor/transform/tables-to-jsx.ts

```diff
@@ -28,6 +28,12 @@ const visitor = (table: Table, index: number, parent: Parents) => {
         ? (cell.children[0] as unknown as Paragraph).children[0]
         : cell.children[0];
 
+    // @note: Compatibility with RDMD. Ideally, I'd put this in a separate
+    // transformer, but then there'd be some duplication.
+    visit(cell, 'break', (_, index, parent) => {
+      parent.children.splice(index, 1, { type: 'text', value: '\n' });
+    });
```

Comment:
so this just converts breaks to a `"\n"` character? (any chance that'll break <kbd>Shift</kbd>+<kbd>Return</kbd> sequences within table cells? or any other potential externalities?)

Reply:
It _shouldn't_, the editor doesn't create `break` nodes explicitly, but leaves it up the the current rendering library to decide how to serialize newlines. This is only an issue because remark < v8 used `break` nodes, but remark >=9 doesn't and just leaves the newlines in place.

This code should only trigger during the migration.